pkgs/test_core/lib/src/executable.dart

```diff
@@ -147,14 +164,16 @@ main(List<String> args) async {
         "with the stack trace and instructions for reproducing the error.");
     exitCode = exit_codes.software;
   } finally {
-    await close();
+    await runner?.close();
   }
 
   // TODO(grouma) - figure out why the executable can hang in the travis
   // environment. https://github.com/dart-lang/test/issues/599
   if (Platform.environment["FORCE_TEST_EXIT"] == "true") {
     exit(exitCode);
   }
+
+  return;
```

Comment:
did something complain before you added this? It looks unnecessary to me.

Reply:
I can't really remember, I think I got a lint before.  I'll put in a PR to clean it up